examples/site/testing_pubsub/pubsub_integration_test.cc

```diff
@@ -42,8 +42,9 @@ class PubsubIntegrationTest : public ::testing::Test {
   void SetUp() override {
     curl_global_init(CURL_GLOBAL_ALL);
 
-    auto const exe =
-        bfs::path(argv0).parent_path() / "pubsub_integration_server";
+    static auto const kPath = std::vector<bfs::path>{
+        bfs::canonical(argv0).make_preferred().parent_path()};
+    auto const exe = bp::search_path("pubsub_integration_server", kPath);
```

Comment:
Should we be consistent about when we encapsulate this as `ExePath()`?

Reply:
Done